src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_people_2, building.end_time_people_2)
+    ]
+    return people_at_times
+
+
+def calculate_internal_load_people(building: Clinic):
+    """
+    Computes the internal heat load of the total people in kWh for a building.
+
+    """
+    p1 = internal_load_people_1(MY_CLINIC)
+    p2 = internal_load_people_2(MY_CLINIC)
+
+    print(p1[i] + p2[i] for i in range(len(p1[0])))
+
+
+calculate_internal_load_people(MY_CLINIC)
+
+
+def internal_load_lighting(building: Clinic):
+    """
+    Computes the internal heat load of the lamps in kWh for a building.
+
+    """
+    lamps_at_times = [
+        [
+            [0, building.lamps_internal * building.wattage_lamps][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(building.start_time_lamps, building.end_time_lamps)
+    ]
+    return lamps_at_times
+
+
+def equipment_load(building: Clinic):
+    """
+    Computes the internal heat load of the fridge in kWh for a building.
+
+    """
+
+    return building.fridge_wattage * building.time_fridge / 1000
+
+
+def infiltration_load(building: Clinic, temperature):
+    """
+    Computes the infiltration load in kWh for a building.
+
+    """
+    infiltration_at_times = [
+        [
+            [
+                0,
+                building.changes
+                * building.volume_cold_store
+                * building.energy_new_air
+                * (temperature - building.inside_ideal_temperature),
+            ][t > start and t < end]
+            for t in range(24)
+        ]
+        for start, end in zip(
+            building.start_time_infiltration, building.end_time_infiltration
+        )
+    ]
+    return infiltration_at_times
+
+
+# infiltration_load(MY_CLINIC)
+
+
+def calculate_cooling_load(building: Clinic, temperature):
+    """
+    Computes the total cooling load in kWh for a building.
+
+    """
+
+    q1 = calculate_transmission_load(MY_CLINIC, temperature)
+    q2 = calculate_internal_load_people(
+        MY_CLINIC,
+    )
+    q3 = internal_load_lighting(MY_CLINIC)
+    q4 = equipment_load(MY_CLINIC)
+    q5 = infiltration_load(MY_CLINIC, temperature)
+
+    print(q1, type(q1))
+    print(q2, type(q2))
+    print(q3, type(q3))
+    print(q4, type(q4))
+    print(q5, type(q5))
+    return (q1 + q2[i] + q3[i] + q4 + q5[i] for i in range(len(q2[0])))
+
+
+def safety_factor_load(building: Clinic, temperature):
+    """
+    Adds a 20% safety factor to the calculation to account for errors and variations from design.
+
+    """
+
+    cooling_load = calculate_cooling_load(MY_CLINIC, temperature)
+
+    return cooling_load * 0.2 + cooling_load
+
+
+def refrigeration_cooling_capacity_sizing(building: Clinic, temperature):
+    """
+    calculating the cooling capacity in kW from the cooling load and the run hours in a day
```

Comment:
### :bug: kWh or kW?
The rest of CLOVER works on an hourly basis, and so it probably makes sense to do that here... I.E., produce an hourly profile in kWh, or, an instantaneously load in kW, so that CLOVER can integrate with this and simply add it to its loads.

If this doesn't make sense, contact me @BenWinchester or any of us really @CLOVER-energy/clover-development-team